client/src/main/scala/com/lnvortex/client/VortexClient.scala

```diff
@@ -658,7 +658,7 @@ case class VortexClient[+T <: VortexWalletApi](vortexWallet: T)(implicit
         for {
           _ <- vortexWallet.broadcastTransaction(signedTx)
           _ <- vortexWallet
-            .labelTransaction(signedTx.txId, s"LnVortex Mix Anon set: $anonSet")
+            .labelTransaction(signedTx.txId, s"LnVortex Mix Anonimity set: $anonSet")
```

Comment:
I'm trying to remove the word `Mix` from the codebase, if you could remove that here, it'd be great :)